Library/Formula/tenfourfox.rb

```diff
@@ -3,24 +3,24 @@
 class Tenfourfox < Formula
   homepage 'http://www.floodgap.com/software/tenfourfox/'
   urlbase = 'https://downloads.sourceforge.net/tenfourfox/'
-  version '31.0.0'
+  version 'fpr32.5'
```

Comment:
Tigerbrew won't recognize this version as newer than 31.0.0; how about we reformat it to `32.5.0-fpr`, so `brew upgrade` works?

Reply:
np, $version was used to construct the url to fetch from based on the CPU arch but I've hardcoded it there.